doc/langref.html.in

```diff
@@ -2861,6 +2861,36 @@ fn dump(args: anytype) void {
     expect(args.b);
     expect(args.s[0] == 'h');
     expect(args.s[1] == 'i');
+}
+      {#code_end#}
+      <p>
+      Anonymous structs without field names may be created, and are referred to as tuples.
```

Comment:
Anonymous structs can be created without specifying field names, and are referred to as "tuples".